sql-statements/sql-statement-drop-sequence.md

```diff
@@ -27,6 +27,10 @@ The `DROP SEQUENCE` statement drops the sequence object in TiDB.
 
```

Comment:
line 18-21 needs to be deleted according to https://github.com/pingcap/docs-cn/pull/3294/files: 

```
**OptTemporary:**
![OptTemporary](/media/sqlgram/OptTemporary.png)

```

Reply:
Thanks for the detailed reviews!